geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/filter/FilterTest.scala

```diff
@@ -95,7 +96,7 @@ class FilterTester extends Specification with TestWithDataStore with LazyLogging
     val filterCount = mediumDataFeatures.count(filter.evaluate)
     val query = new Query(sftName, filter)
     Option(projection).foreach(query.setPropertyNames)
-    val queryCount = fs.getFeatures(query).size
+    val queryCount = SelfClosingIterator(fs.getFeatures(query)).length
```

Comment:
actually, even with the query hint it will use the stats for counting, which should be 100% correct, but probably safest just to actually count the length.